ReactAndroid/src/androidTest/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
           package="com.facebook.react.tests.gradle"
           android:versionCode="1"
           android:versionName="1.0" >
-  <uses-sdk android:targetSdkVersion="7" />
+  <uses-sdk android:targetSdkVersion="27" />
```

Comment:
This is useless.

Reply:
should be completely removed. Also libraries should not even define `targetSdk`